.travis.yml

```diff
@@ -36,6 +40,10 @@ jobs:
     # ---------------------------------------
     - stage: lint
       name: 'Linter'
+      if: (type IN (pull_request))
+        AND (branch IN (master, 4-dev, 3-dev, 2-dev, 2-stable))
```

Comment:
Aren't these 2 lines redundant with line 29? By the way, the `type` condition is not the same: `pull_request` here, VS `cron|pull_request`.

I would rather delete these 2 and just keep the `skip-ci|wip` condition.

Reply:
Delete the whole condition IMO
@xbill82 about skipping CI build in Travis https://github.com/kuzzleio/kuzzle-admin-console/pull/854#discussion_r535090753